guides/common/modules/proc_changing-the-order-of-ansible-roles.adoc

```diff
@@ -5,6 +5,6 @@ Use the following procedure to change the order of Ansible roles applied to a ho
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts > All Hosts*.
-. O the *Ansible* tab, click the *Edit Ansible Roles* button.
-. In the *Edit Ansible Roles* window you can change the order of the roles by dragging and dropping the roles into the preferred position.
-. Click *Confirm* to change the order of the Ansible roles.
+. Select the *Ansible Roles* tab.
```

Comment:
```suggestion
. Select a host.
. Select the *Ansible Roles* tab.
```
I think we're missing a step or two here.

It's hosts > all hosts > selecting a host > select "Ansible" tab > by default, you are on the "Roles" tab.

(tested on Foreman 3.3.0)